.github/workflows/main_workflow.yml

```diff
@@ -213,17 +213,11 @@ jobs:
           docker pull trufflesuite/ganache:beta
 
           # build required images
-          pip install .
-          autonomy fetch --local --service valory/hello_world
 
-          cd hello_world
-          autonomy build-images
-          autonomy build-images --dev
-          autonomy build-images --dependencies
-          autonomy build-images --dependencies --version 1.0.0
-          autonomy deploy build ../deployments/keys/hardhat_keys.json
-
-          cd ..
+          skaffold build --push=false
+          skaffold build -p release --push=false
```

Comment:
These builds take a long time...so we should at least build with cache no? https://github.com/valory-xyz/open-autonomy/runs/8025408228?check_suite_focus=true 

Which means we need to have a specific scaffold flag just for the CI

Reply:
We just need it until the release, once we perform the release and publish these images we'll just pull them in the CI.

Reply:
Good point

Reply:
Please open an issue so we don't forget

Reply:
https://github.com/valory-xyz/open-autonomy/issues/1234